tests/python/pants_test/init/test_plugin_resolver.py

```diff
@@ -6,7 +6,7 @@
 from contextlib import contextmanager
 from pathlib import Path, PurePath
 from textwrap import dedent
-from typing import Iterable
+from typing import Dict, Iterable, Optional
```

Comment:
No meaningful changes to this file, outside of needing to add two trivial type annotations.